src/pages/our-principles/_data.tsx

```diff
@@ -39,6 +38,7 @@ export const principles: Array<PrinciplesType> = [
             },
         ],
         img: Reliable,
+        img_alt: '_t_Deriv values reliability_t_',
```

Comment:
we don't need to translate `alt` property of image

Reply:
@NikitK-deriv as per the SEO recommendation we need to translate `alt` as well. Plus, translations `alt` texts are done as well.
https://deriv-group.slack.com/archives/C01HSQRH7AM/p1681823347069359